roles/common/files/sensu_plugins/metrics-neutron-fip.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
Seems to be a lot of code duplication between this one and check-neutron-fip. Is there any way to be more DRY?